MANIFEST.md

```diff
@@ -8,15 +8,15 @@ author:
 copyright_owner:
   - Stefan Wunsch
   - Communications Engineering Lab (CEL) at Karlsruhe Institute of Technology (KIT)
-dependencies (most of them are needed by GNU Radio 3.8 and are included transitively):
-  - gnuradio maint-3.8
+dependencies (most of them are needed by GNU Radio 3.10 and are included transitively):
+  - gnuradio maint-3.10
   - UHD = 3.15
   - QT 5.10.1
   - Qwt = 6.1.0
   - python-matplotlib
 repo: https://github.com/kit-cel/gr-radar
 website: https://grradar.wordpress.com
-gr_supported_version: v3.7, v3.8
+gr_supported_version: v3.7, v3.8, v3.10
```

Comment:
Are all versions supported at the same time? I'd think that 3.7 & 3.8 won't work after the update to 3.10?

Reply:
Well... Only 3.10 will work on this branch, but the repo has branches for maint-3.7 and maint-3.8. Currently CGRAN only displays the info from the default branch. Including all the available supported versions in the manifest is the current "standard practice". Improving/changing this is a CGRAN level discussion. The Readme, more relevant to people viewing the repo and source code only mentions 3.10 support.

Reply:
I.e.
https://github.com/daniestevez/gr-satellites/blob/036034a6cd03aef219614c53dd779419da625051/MANIFEST.md?plain=1#L14